docs/pages/en/guide/concepts.html

```diff
@@ -16,8 +16,8 @@
 The requirements for a video editing professional's workstation are different from the kid's homework PC.  For your home computers,
 it might not matter very much if you overpay by $50 for the same computer, or a similar one (even if that's never fun).</p> <p>If you need to purchase PCs for an entire school district, overpaying by $50 per PC can break the budget.  But just like purchasing
 for your home, it can be hard or impossible to sift through the dozens of vendors, dozens of manufacturers,
-and hundreds of configurations to find exactly the right computer at the lowest price.</p> <h2 id="workload"><a href="#workload" aria-hidden="true" class="header-anchor">#</a> Workload</h2> <p>Say that two of your hobbies video editing and reading e-books. Clearly, each of these activities will require different
-resources from your computer.  Home computers have highly-variable workloads, which is why
+and hundreds of configurations to find exactly the right computer at the lowest price.</p> <h2 id="workload"><a href="#workload" aria-hidden="true" class="header-anchor">#</a> Workload</h2> <p>Say that two of your hobbies involve video editing and reading e-books. Clearly, each of these activities will require different
```

Comment:
You need to change these in the correct `.md` file, and follow the directions in the wiki to get everything set up to PR.